arangograph/certificates.md

```diff
@@ -0,0 +1,150 @@
+  The communication with and between the servers of an ArangoGraph deployment is
+  encrypted using the TLS protocol
+---
+# X.509 Certificates
+
+{{ page.description }}
+{:class="lead"}
+
+X.509 certificates are digital certificates that are used to verify the
+authenticity of a website, user, or organization in a public key infrastructure
+(PKI). They are used in various applications, including SSL/TLS encryption,
+which is the basis for HTTPS - the primary protocol for securing communication
+and data transfer over a network.
+
+The X.509 certificate format is a standard defined by the
+[International Telecommunication Union (ITU)](https://www.itu.int/en/Pages/default.aspx){:target="_blank"}
+and contains information such as the name of the certificate holder, the public
+key associated with the certificate, the certificate's issuer, and the
+certificate's expiration date. An X.509 certificate can be signed by a
+certificate authority (CA) or self-signed.
+
+ArangoGraph is using:
+- **well known X.509 certificates** created by
+[Let's Encrypt](https://letsencrypt.org/){:target="_blank"}
+- **self-signed X.509 certificates** created by ArangoGraph platform
+
+## Certificate chains
+
+A certificate chain, also called the chain of trust, is a hierarchical structure
+that links together a series of digital certificates. The trust in the chain is
+established by verifying the identity of the issuer of each certificate in the
+chain. The root of the chain is a trusted third-party, such as a certificate
+authority (CA). The CA issues a certificate to an organization, which in turn
+can issue certificates to servers and other entities. 
+
+For example, when you visit a website with an SSL/TLS certificate, the browser
+checks the chain of trust to verify the authenticity of the digital certificate.
+The browser checks to see if the root certificate is trusted, and if it is, it
+trusts the chain of certificates that lead to the end-entity certificate.
+If any of the certificates in the chain are invalid, expired, or revoked, the
+browser will not trust the digital certificate.
+
+## X.509 certificates in ArangoGraph
+
+Each ArangoGraph deployment is accessible on two different port numbers:
+- default port `8529`
+- high port `18529`
+
+Each ArangoGraph Notebook is accessible on two different port numbers:
+- default port `8840`
+- high port `18840`
+
+Metrics are accessible on two different port numbers:
+- default port `8829`
+- high port `18829`
+
+The distinction between these port numbers is in the certificate used for the
+TLS connection.
+
+### Well known X.509 certificates
+
+**Well known X.509 certificates** created by
+[Let's Encrypt](https://letsencrypt.org/){:target="_blank"} are used on the
+default ports, i.e. `8529`.
+
+This type of certificate has a lifetime of 5 years and is rotated automatically.
+It is recommended to use well known certificates, as this eases the access of a
+deployment in your browser.
```

Comment:
IMO we should say it's a wildcard certificate, and so cannot contain SANs (as can be specified when using PE)

Reply:
Note that if you create a certificate where the 'well-known' switch is turned off, both ports serve the identical self-signed certificate.

Reply:
done.